src/sagemaker/tuner.py

```diff
@@ -73,22 +73,13 @@ class ContinuousParameter(_ParameterRange):
     """
     __name__ = 'Continuous'
 
-    def __init__(self, min_value, max_value):
-        """Initialize a ``ContinuousParameter``.
-
-        Args:
-            min_value (float): The minimum value for the range.
-            max_value (float): The maximum value for the range.
-        """
```

Comment:
would it be useful to still have the types in the docstrings noted somewhere? the base class says int or float but both subclasses here require one or the other and not both.